databases/week3/DB_Week3.sql

```diff
@@ -0,0 +1,140 @@
+);
+
+INSERT INTO meal (title, meal_description, meal_location, accessibility_time, max_reservations, price, created_date)
+VALUES ('Beef tartare', 'Beef with something that I hear first time in my life', 'Copenhagen', '2022-07-10 15:00:00', 50, 200, '2022-06-20'),
+        ('Salad with mozzarella', 'Just some grass with some cheese', 'Odense', '2022-07-07 18:00:00', 30, 100, '2022-06-25'),
+        ('Shripm with pear', 'Shripm with pear and sweet sauce', 'Nykobing F', '2022-07-20 14:00:00', 20, 150, '2022-07-02' );
+
+INSERT INTO reservation (number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email)
+VALUES (2, 1, '2022-06-21', '91546755', 'Johan', 'johanS@gmail.com'),
+        (5, 1, '2022-06-22', '75342233', 'Nikolas', 'nicolasP@gmail.com'),
+        (8, 2, '2022-06-26', '34556678', 'Lillian', 'lillianV@gmail.com'),
+        (4, 2, '2022-06-27', '23445565', 'Thomas', 'thomasK@gmail.com'),
+        (10, 3, '2022-07-03', '44553455', 'Jane', 'janeJ@gmail.com'),
+        (10, 3, '2022-07-05', '12334455', 'Thor', 'thorO@gmail.com');
+
+INSERT INTO review (title, review_description, meal_id, stars, created_date)
+VALUES ('Good', 'Good meal, very good meal', 3, 5, '2022-07-02'),
+        ('Bad', 'Dry meat', 1, 1, '2022-06-21'),
+        ('OK', 'Average vagetables quality', 2, 3, '2022-06-27'),
+        ('Amazing', 'Very good vagetables quality', 2, 5, '2022-06-28'),
+        ('The best', 'Meat is perfect', 1, 5, '2022-06-22'),
+        ('Little bit disappointed', 'Food was food, but too cold', 1, 2, '2022-06-23');
+
+-- Meal Queries
+SELECT * FROM meal;
+
+INSERT INTO meal (title, meal_description, meal_location, accessibility_time, max_reservations, price, created_date)
+VALUES ('Melon horned with ham', 'I can not imagine who has created this meal', 'Stege', '2022-07-6 16:00:00', 25, 120, '2022-07-03');
+
+SELECT * FROM meal
+WHERE id = 1;
+
+UPDATE meal
+SET title = 'Beef and beef', price = 210
+WHERE id = 1;
+
+DELETE FROM meal
+WHERE id = 4;
+
+-- Reservation Queries
+SELECT * from reservation;
+
+INSERT INTO reservation (number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email)
+VALUES (3, 2, '2022-06-23', '91546754', 'Marta', 'martaQ@gmail.com');
+
+SELECT * from reservation 
+WHERE id = 1;
+
+UPDATE reservation
+SET contact_phonenumber = 34334455, contact_name = 'Soren'
+WHERE id = 1;
+
+DELETE FROM reservation
+WHERE id = 1;
+
+-- Review Queries
+SELECT * FROM review;
+
+INSERT INTO review (title, review_description, meal_id, stars, created_date)
+VALUES ('Shocking', 'The weather was too bad, but the meal was good', 2, 5, '2022-06-28');
+
+SELECT * FROM review
+WHERE id = 1; 
+
+UPDATE review
+SET title = 'Unexpected good', review_description = 'Perfect', stars = 5
+WHERE id = 1;
+
+DELETE FROM review
+WHERE id = 1;
+
+-- Additional Queries
+SELECT * FROM meal
+WHERE price < 150;
+
+SELECT meal.title, meal.max_reservations, COUNT(reservation.id) AS Reserved_number FROM meal
```

Comment:
This is the tricky query. Bottom line, you need to sum the number of guests for all reservations for a given meal id. One option is to use the SQL function COALESCE, but there are other ways too :)